configuration/reissue-internal-certificates.adoc

```diff
@@ -24,19 +25,22 @@ include::modules/reissue-internal-certificates-scanner.adoc[leveloffset=+1]
 //Restart Scanner & Scanner DB
 include::modules/restart-scanner-and-scannerdb-containers.adoc[leveloffset=+2]
 
-[id="reissue-internal-certificates-secured-clusters"]
-== Reissuing internal certificates for Sensor, Collector, and Admission controller
-Sensor, Collector, and Admission controller use certificates to communicate with each other, and with Central.
+[id="reissue-internal-certificates-secured-clusters_{context}"]
+== Reissuing internal certificates for secured clusters
+Secured clusters contain the Collector, Sensor, and Admission Control components.
+These components communicate with each other, and with Central by using certificates.
 
-To replace the certificates, use one of the following methods:
+Choose the appropriate method to replace the certificates:
 
 * Create, download, and install an init bundle on the secured cluster. You must have the `Admin` user role to create an init bundle.
-* Use the automatic upgrades functionality. Automatic upgrades are available only for static manifest deployments using the `roxctl` CLI.
-//Add link to automatic upgrades
+* Use the automatic upgrades feature, which is only available for static manifest deployments by using the `roxctl` CLI.
```

Comment:
The new Secured Cluster auto certificate renewal feature is not related to "automatic upgrades feature".

As of 4.7.0, there are 3 ways of renewing SC certificates:
* [new] automatic certificate renewal (available for Operator & Helm installs)
* create, download and install an init bundle - this is the old way for Operator & Helm installs and should not be recommended anymore (except if the certificates are already expired and the Secured Cluster cannot connect to Central anymore)
* using the automatic upgrades feature (for legacy manifest-based deployments that use roxctl) - this is a legacy method from the Stackrox days and RH customers typically don't use it